src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -771,6 +772,9 @@ You can opt to place this file in `$PROACTIVE_HOME/config/authentication/azure.c
 * *_Shared Image Gallery:_* The prefix `gallery#` allows you to use images from an existing Azure https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries[Shared Image Gallery]. Append the name of your Resource Group, your Gallery and your Image as `resourceGroup:gallery:image`. Optionally, you can specify a version too (`:version`); if no version is given, the latest image version will be used by ProActive. For example: +
 `gallery#proactive-rg:activeeon-gallery:my-image` +
 `gallery#myRG:myGallery:myImage:1.0`
+** *_Shared Image Gallery from non-default subscription_* It is possible to use images from an existing Azure https://docs.microsoft.com/en-us/azure/virtual-machines/linux/shared-image-galleries[Shared Image Gallery] from the specific (different from default) subscription. +
```

Comment:
- You can use ... (avoiding passive form)
- 'from the specific' => by manually specifying the subscription